MyWeightTests/Snapshot+Extensions.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  Snapshot+Extensions.swift
+//  MyWeight
+//
+//  Created by Bruno Mazzo on 11/2/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import Foundation
+import Nimble
+import Nimble_Snapshots
+
+public struct ScreenSnapshot {
+    let name: String
+    let record: Bool
+    let size: CGSize
+    
+    init(name: String, record: Bool, size: CGSize) {
+        self.name = name
+        self.record = record
+        self.size = size
+    }
+}
+
+public func snapshotScreen(_ name: String, sizes: [String: CGSize]) -> [ScreenSnapshot] {
```

Comment:
I feel a bit uncomfortable with global functions.
What do you think to create a `SnapshotService` struct with them?